Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutAnalytics.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+class BTPayPalNativeCheckoutAnalytics {
+    
+    // MARK: - Tokenize Events
+    
+    // Counted in conversion rates
+    static let vaultRequestStarted = "paypal-native:vault-tokenize:started"
+    static let checkoutRequestStarted = "paypal-native:checkout-tokenize:started"
+    static let tokenizeFailed = "paypal-native:tokenize:failed"
+    static let tokenizeSucceeded = "paypal-native:tokenize:succeeded"
+    static let tokenizeCanceled = "paypal-native:tokenize:canceled"
+    
+    // Specific tokenize failures
+    static let tokenizeUrlRequestFailed = "paypal-native:tokenize:url-request:failed"
+    static let tokenizeParsingResultFailed = "paypal-native:tokenize:parsing-result:failed"
+    
+    // MARK: - Create Order Events
+    
+    static let createOrderStarted = "paypal-native:create-order:started"
+    static let createOrderFailed = "paypal-native:create-order:failed"
+    static let createOrderSucceeded = "paypal-native:create-order:succeeded"
+    
+    // Specific Create Order Fails
+    static let createOrderPayPalNotEnabledFailed = "paypal-native:create-order:paypal-not-enabled:failed"
```

Comment:
I wasn't sure if we needed to keep these detailed messages.

Reply:
I think we can remove the more specific portion and just do `paypal-native:create-order:failed`